addons/gmc-toolbox/toolbox_dock.tscn

```diff
@@ -54,7 +51,7 @@ text = "MPF lights: Config File"
 [node name="edit_mpf_lights_config" type="LineEdit" parent="MainVContainer/TopHContainer/LeftVContainer/container_mpf_lights_config"]
 layout_mode = 2
 size_flags_horizontal = 3
-text = "/Users/Anthony/Git/bmo-game/config/config_lights.yaml"
+text = "C:/Users/carr-/Documents/development/darkchaos/glf_mpf/config/lights.yaml"
```

Comment:
Ugh, I hate how Godot saves the value from a project when a plugin is being used. I wish there was a way to block this and preserve an empty string.